.env

```diff
@@ -0,0 +1,3 @@
+CACHE_MODE=
```

Comment:
Can we reuse the existing config.json file instead?

Reply:
This is what I think.

Reply:
There are quite a bit of reusable codes in cache.js like `accumulateContent`, `parsingContent`. How about modularize these codes in a separate file, then separate out the implementation of middleware as you said? So that we do not have to modify the codes in all the implementations if you change some logic in things like `accumulateContent` . 

The structure will be like
```
cache-reusable-logic.js
google-datastore.js
elasti-cache.js
```

in the main.js will be like
```
if (cacheMode == 'elastiCache')
   elastiCache.middleware()
if (cacheMode == 'google-datastore')
   google-datastore.middleware()
```

Reply:
As for the CLA thing, I have actually signed the agreement. Is there some other steps I might have missed?